js/bittrex.js

```diff
@@ -700,15 +700,14 @@ module.exports = class bittrex extends Exchange {
         const isCeilingOrder = isCeilingLimit || isCeilingMarket;
         if (isCeilingOrder) {
             request['ceiling'] = this.priceToPrecision (symbol, price);
-            request['timeInForce'] = 'IMMEDIATE_OR_CANCEL';
         } else {
             request['quantity'] = this.amountToPrecision (symbol, amount);
-            if (uppercaseType === 'LIMIT') {
-                request['limit'] = this.priceToPrecision (symbol, price);
-                request['timeInForce'] = 'GOOD_TIL_CANCELLED';
-            } else {
-                request['timeInForce'] = 'IMMEDIATE_OR_CANCEL';
-            }
+        }
+        if (isCeilingMarket || uppercaseType === 'MARKET') {
+            request['timeInForce'] = 'IMMEDIATE_OR_CANCEL';
+        } else {
+            request['limit'] = this.priceToPrecision (symbol, price);
+            request['timeInForce'] = 'GOOD_TIL_CANCELLED';
```

Comment:
A `CEILING_LIMIT` order cannot be have timeInForce GTC:

- https://bittrex.github.io/api/v3

<img width="1314" alt="Screen Shot 2020-03-30 at 23 14 28" src="https://user-images.githubusercontent.com/1294454/77957505-425d3e00-72dc-11ea-8675-18cbc2849b15.png">

Reply:
> a limit ceiling order got 'IMMEDIATE_OR_CANCEL' set, because it went into the first branch.

That's actually the most reasonable value to set, because FOK is even worse.

Reply:
Let me know if that does not explain it, and if I am missing something. To my mind, we should close this PR in the light of the above.

Reply:
@kroitor 
Interesting, thank you!
Is my thought correct, that a (ceiling) limit order should be GOOD_TIL_CANCELLED?
I updated the PR to document this, close or merge.